react/introduction/fundamentals/react-what-is-state.md

```diff
@@ -47,7 +47,7 @@ class Hello extends React.Component {
 }
 ```
 
-This will throw an error, however, because `this.state` is `null` at the moment. The error would be `Cannot read property 'x' of null`.
+This will throw an error, however, because `this.state` is `undefined` at the moment. The error would be `Cannot read property 'x' of undefined`.
 
 To set the **initial state**, by giving `text` a value, you can make use of `constructor()`. This method is a special function that gets executed when the component is created (creating an instance of the `React.Component` class). We'll look at these in depth in the future workouts.
```

Comment:
```suggestion
To set the **initial state**, by giving `text` a value, you can make use of `constructor()`. This method is a special function that gets executed when the component is created (creating an instance of the `React.Component` class). We'll look at these in-depth in future workouts.
```

Reply:
this is good review, we definitely don't need `'the'` there. as for `'in depth'`, I am refering to the phrase/idiom not the adjective meaning that it is correct in this case

https://ell.stackexchange.com/a/102707